### PR TITLE
Add a Type definition for uiType used in the text-editor

### PR DESCRIPTION
**Pull request by john-traas**
Fixes https://github.com/Lundalogik/crm-feature/issues/4455

May need separate PR's in both building blocks and lime-crm-components to then import and use the Type definition.

### CODE REVIEW

File: src/components/text-editor/types.ts [hunk 1]
```diff
@@ -7,3 +7,5 @@ import { CustomElementDefinition } from '../../global/shared-types/custom-elemen
 export type TextEditorPlugin = {
     node: CustomElementDefinition[];
 };
+
+export type EditorUiTypes = 'standard' | 'minimal';
```
Comment (hannahu) [inline src/components/text-editor/types.ts:11]: Singular ☺️ 

```suggestion
export type EditorUiType = 'standard' | 'minimal';
```

Comment (john-traas) [reply src/components/text-editor/types.ts]: Updated

File: src/components/text-editor/types.ts [hunk 2]
```diff
@@ -7,3 +7,5 @@ import { CustomElementDefinition } from '../../global/shared-types/custom-elemen
 export type TextEditorPlugin = {
     node: CustomElementDefinition[];
 };
+
+export type EditorUiType = 'standard' | 'minimal';
```
Comment (jgroth) [inline src/components/text-editor/types.ts:11]: There's a missing release tag on this type

**Review by hannahu — APPROVED** [review]

**Review by jgroth — APPROVED** [review]

**Pull request merged**